src/npc.cpp

```diff
@@ -717,7 +718,7 @@ int NpcScriptInterface::luagetDistanceTo(lua_State* L)
 int NpcScriptInterface::luaSetNpcFocus(lua_State* L)
 {
 	// doNpcSetCreatureFocus(cid)
-	Npc* npc = tfs::lua::getScriptEnv()->getNpc();
+	auto npc = tfs::lua::getScriptEnv()->getNpc();
```

Comment:
Could this be `const auto`
```suggestion
	const auto npc = tfs::lua::getScriptEnv()->getNpc();
```